eox_tagging/api/v1/test/test_permissions.py

```diff
@@ -0,0 +1,49 @@
+""" Test classes for API Permissions. """
+from django.contrib.auth.models import Permission, User
+from django.test import TestCase, override_settings
+from mock import Mock
+
+from eox_tagging.api.v1.permissions import EoxTaggingAPIPermission, load_permissions
+
+
+@override_settings(EOX_TAGGING_LOAD_PERMISSIONS=True)
+class TestAPIPermissions(TestCase):
+    """Test class for tags viewset."""
+
+    def setUp(self):
+        """ Permissions setup."""
+        load_permissions()
+
+        # User with permission to acccess the API
+        user_permission = Permission.objects.get(codename="can_call_eox_tagging")
+        self.user_authorized = User.objects.create(username="user_with_permission")
+        self.user_authorized.user_permissions.add(user_permission)
+
+        # Common user without permission
+        self.common_user = User.objects.create(username="user_without_permission")
+
+        self.has_permission = EoxTaggingAPIPermission().has_permission
+
+    def test_API_access_success(self):
```

Comment:
here you are testing only two cases, but you can test more in order to ensure  the behavior,
1. user has the permission and get_host() returns the same url (This test)
2. user doesn have any permissions and get_host() returns the same url (test line 39)
3. user has the permission and get_host() doesn't return the same url
4. user is staff and get_host() returns the same url.
5. user is staff and get_host() doesn't return the same url
6. get host raises an exception

and you can test more things like  mock_request.get_host.assert_called_once(), in order to verify that get_host was called

Reply:
You're right. I'm on it

Reply:
I'm not going to add the is_staff tests because as I understand we are going to delete that from the permissions